common/lib/xmodule/xmodule/tests/test_services.py

```diff
@@ -4,16 +4,23 @@
 
 from __future__ import absolute_import
 
-from unittest import TestCase
+import unittest
+from django.test import TestCase
 
 import ddt
 import mock
+
 from config_models.models import ConfigurationModel
 from django.conf import settings
 from django.test.utils import override_settings
 from xblock.runtime import Mixologist
 
-from xmodule.services import ConfigurationService, SettingsService
+from opaque_keys.edx.locator import CourseLocator
+from xmodule.services import ConfigurationService, SettingsService, TeamsConfigurationService
+from xmodule.modulestore.tests.factories import CourseFactory
+from openedx.core.lib.teams_config import TeamsConfig
+from xmodule.partitions.partitions import Group, UserPartition
+from fs.memoryfs import MemoryFS
```

Comment:
Last nit: I don't think these imports are used anywhere.  You could try running `isort` on this file to clean it up (actually, I'm surprised isort didn't run automatically as part of the jenkins quality build).